### DISCUSSION TITLE
Inherit / show achievements from my organization's repositories

### DISCUSSION THREAD

**Discussion by MrCsabaToth** [community/community#63608] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The achievements seem to feed only from the repositories of my user. Most of my work is separated into organizations, so each project can have their own website and encapsulate the related repositories under organizations. I'm the owner of those repositories and either I'm the sole team member, or there are some contributors / team members but I'm the main contributor by far. This is especially noticeable with the Starstruck achievement.

It'd be great to aggregate the achievements in my organizations into my user's achievements. The achievements displayed for my user are from the past, when I didn't use organizations.

**Comment by Vicwxy9** [comment]
https://github.com/4xmen/Get-Github-Achievements follow this link and you will get the guided steps for it

**Reply by MrCsabaToth** [reply]
> https://github.com/4xmen/Get-Github-Achievements follow this link and you will get the guided steps for it

Starstruck achievement doesn't require any steps, supposed to be automatic. "You can get this Achievement When a Repository on your account hits 16 stars, even if it was transferred, you will receive the badge."

The 'how to get' button didn't help either. I didn't find docs on how to include my organizations badges into my profile. Can you point it out specifically?

**Comment by rainbow1016** [comment]
very good question!
